src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -133,9 +133,10 @@ var inputValueTracking = {
       return false;
     }
     var tracker = getTracker(subject);
+    const isNode = (subject: any).nodeType === ELEMENT_NODE;
 
     if (!tracker) {
-      if (typeof (subject: any).tag === 'number') {
+      if (isNode || typeof (subject: any).tag === 'number') {
```

Comment:
I think @aweary is saying that if `subject` is a node itself then `subject.stateNode` (which is what we pass to `trackNode`) is going to be `undefined`.